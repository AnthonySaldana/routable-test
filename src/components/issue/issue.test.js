import React from 'react';
import ReactDOM from 'react-dom';
import Issue from '../issue';

const mockProps = {
    issue: {
        title: "Test Issue"
    }
};

let elem;

beforeEach(() => {
    elem = document.createElement('div');
    ReactDOM.render(<Issue issue={mockProps.issue} />, elem);
});

it('Matches Snapshot', () => {
  expect(elem).toMatchSnapshot();
});

it('Issue Name matches up', () => {
    expect(elem.getElementsByClassName('issueCard')[0].innerHTML).toEqual('Test Issue');
});