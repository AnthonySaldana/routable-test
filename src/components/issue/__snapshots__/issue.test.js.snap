// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Snapshot 1`] = `
<div>
  <div
    class="issueCard"
    draggable="true"
  >
    Test Issue
  </div>
</div>
`;
